example/src/Benchmarks.ts

```diff
@@ -14,6 +14,7 @@ export const benchmarkAgainstAsyncStorage = async () => {
     const start = global.performance.now();
     for (let i = 0; i < iters; i++) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      // @ts-ignore
```

Comment:
why?

Reply:
Originally, there was an issue with webpack not working since unused variables were not allowed in the original typescript set-up. This has been resolved with changing the main tsconfig to allow unused variables.

Reply:
Originally, there was an issue with webpack not working since unused variables were not allowed in the original typescript set-up. This has been resolved with changing the main tsconfig to allow unused variables.